NuRadioMC/simulation/simulation.py

```diff
@@ -306,14 +306,22 @@ def __init__(self, inputfilename,
         if(Tnoise is not None and Vrms is not None):
             raise AttributeError(f"Specifying noise temperature (set to {Tnoise}) and Vrms (set to {Vrms} is not allowed.")
         if(Tnoise is not None):
-            self._Tnoise = float(Tnoise)
+            if(Tnoise == "detector"):
+                self._Tnoise = None  # the noise temperature is defined in the detector description
+            else:
+                self._Tnoise = float(Tnoise)
```

Comment:
I just noticed here -- sorry. But TNoise is not according to our coding convention, no? should it not be noise_temp or something?

Reply:
hmm this is how it is named in the config file. I would not change it as this will break all existing simulation settings. I could rename it just internally...

Reply:
Is this a 'no' @cg-laser ?